awx/main/models/credential/__init__.py

```diff
@@ -1111,6 +1111,12 @@ def create(self):
                 'secret': True,
                 'help_text': ugettext_noop('A password or token used to authenticate with'),
             },
+            {
+                'id': 'verify_ssl',
+                'label': ugettext_noop('Verify SSL'),
+                'type': 'boolean',
+                'default': True,
```

Comment:
While testing this, I found a bug w/ the ui form code. The bug isn't directly related to this PR but anyone that tries to change the value of this flag will most likely encounter it. https://github.com/ansible/awx/issues/9955

Reply:
https://github.com/ansible/awx/pull/9960